### ISSUE TITLE
Support multiple offsets in dumpfiles

### ISSUE THREAD

**Issue by atcuno** [volatilityfoundation/volatility3#1319]
In many situations, an individual file is cached multiple times, leading filescan to report several addresses. Right now, Volatility takes in `--virtaddr` and `--physaddr` as an IntRequirement. We need to switch this to a ListRequirement like other plugins and then adjust the code to extract from each offset.

**Comment by ikelos** [comment]
Happy for them to become lists, but I think previous we only really accept one (maybe one of each?)  This time, everything seems to be going in the same list and it worries me a little (even just from a clarity-of-code perspective).  Even if it only allows one type or the other, we should leave a big fat comment where they're populated to say how it works.

**Comment by ikelos** [comment]
Yeah, this plugin needs a little bit of work.  It wasn't so bad when it only accepted one offset, but it should process both lists (rather than `elif`) and it should pull the layers out ready in variables before it goes into the offset loop...
